lib/fog/kubevirt/compute/requests/get_server.rb

```diff
@@ -4,21 +4,42 @@ class Compute
       class Real
         def get_server(name)
           vm = get_raw_vm(name)
-          populate_runtime_info(vm)
+          vmi = runtime_vm(vm)
+
+          populate_runtime_info(vmi)
```

Comment:
I don't understand this - populate_runtime_info is expecting only vmi, but inside it, it refers to vm  - which will lead to exception.

I think that we must have a test for the server/servers model to make sure the code is complete.

Reply:
New recording and the spec pushed.